src/SFA.DAS.EmployerIncentives.Domain/ApprenticeshipIncentives/Events/PaymentsPaused.cs

```diff
@@ -0,0 +1,43 @@
+﻿using SFA.DAS.EmployerIncentives.Abstractions.Logging;
+using SFA.DAS.EmployerIncentives.Domain.ApprenticeshipIncentives.Models;
+using SFA.DAS.EmployerIncentives.Domain.IncentiveApplications;
+using SFA.DAS.EmployerIncentives.Domain.ValueObjects;
+
+namespace SFA.DAS.EmployerIncentives.Domain.ApprenticeshipIncentives.Events
+{
+    public class PaymentsPaused : IDomainEvent, ILogWriter
+    {
+        public long AccountId { get; }
+        public long AccountLegalEntityId { get; }
+        public ApprenticeshipIncentiveModel Model { get; }
+        public ServiceRequest ServiceRequest { get; }
+
+        public PaymentsPaused(
+            long accountId,
+            long accountLegalEntityId,
+            ApprenticeshipIncentiveModel model,
+            ServiceRequest serviceRequest)
+        {
+            AccountLegalEntityId = accountLegalEntityId;
+            AccountId = accountId;
+            Model = model;
+            ServiceRequest = serviceRequest;
+        }
+
+        public Log Log
+        {
+            get
+            {
+                var message =
+                    $"Apprenticeship Incentive has been paused for ApprenticeshipIncentiveId {Model.ApplicationApprenticeshipId} and " +
```

Comment:
Not strictly true.  It's OnProcessing or OnError so attempting to pause.  There is an OnProcessed Log hander that would make more sense.